### DISCUSSION TITLE
DatePickerInput

### DISCUSSION THREAD

**Discussion by rishav-brigu** [mantinedev/mantine#3727] [Q&A]
I want to change the color of the selected dates along with the start date and end date and the color of the range between those selected dates in the "DatePickerInput" component. Can anyone please help me regarding this? What styles api should I be using? I searched a lot but I am not getting any solutions.

**Comment by rtivital** [comment]
You can change color of value with styles api. Changing styles of different value parts is not supported.

**Reply by rishav-brigu** [reply]
Thank you so much for the prompt response.

**Reply by bastiengrignon** [reply]
I'm not able to find the correct styles API to change the color of the selected dates ! Have you find you're way to do it ?

**Reply by rtivital** [reply]
Inspect elements in dev tools to find styles api names, they all look like `.mantine-ComponentName-selector`
